packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -233,4 +241,75 @@ export class HttpApi extends Resource implements IHttpApi {
       integration: options.integration,
     }));
   }
+
+  /**
+   * Return the given named metric for this HTTP Api Gateway
+   *
+   * @default Average over 5 minutes
+   */
+  public metric(metricName: string, props?: MetricOptions): Metric {
+    return new Metric({
+      namespace: 'AWS/ApiGateway',
+      metricName,
+      dimensions: { apiName: this.httpApiName },
```

Comment:
From what I see here - https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-metrics.html - apiName is not a dimension. It should be `apiId`.

If that's correct, also revert the changes for API name above.

Reply:
I have left httpApiName property and fixed dimensions